src/components/registration/steps.jsx

```diff
@@ -324,7 +324,7 @@ module.exports = {
         },
         getYearOptions: function () {
             return Array.apply(null, Array(100)).map(function (v, id) {
-                var year = 2016 - id;
+                var year = 2017 - id;
```

Comment:
We shouldn't have hardcoded the year like this — would you please update this to get the current year?